server/cli/accounting.go

```diff
@@ -133,7 +136,13 @@ func runSetAcct(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	server.RunSetAcct(Context, args[0], args[1])
+	// Read in the config file.
+	body, err := ioutil.ReadFile(args[1])
+	if err != nil {
+		log.Errorf("unable to read accounting config file %q: %s", args[1], err)
```

Comment:
yeah, I prefer to gracefully log an error on cli stuff, stack traces tend to be distracting.